hadoop-ozone/dist/src/main/smoketest/basic/ozone-shell.robot

```diff
@@ -111,9 +111,9 @@ Test Bucket Acls
 Test key handling
     [arguments]     ${protocol}         ${server}       ${volume}
                     Execute             ozone sh key put ${protocol}${server}/${volume}/bb1/key1 /opt/hadoop/NOTICE.txt
-                    Execute             rm -f NOTICE.txt.1
-                    Execute             ozone sh key get ${protocol}${server}/${volume}/bb1/key1 NOTICE.txt.1
-                    Execute             ls -l NOTICE.txt.1
+                    Execute             rm -f /tmp/NOTICE.txt.1
```

Comment:
Sorry, I don't understand what is happening here. Did we remap /opt to /tmp somehow? and the file name is also remapped ?

Reply:
To make the acceptance test execute simple I start the acceptance tests directly from the VM  (uid=1001) and not from docker container (uid=1000). But the build has been done with uid=1000. To make it possible to run I use /tmp for writing/reading temporary files which is always writable.